.github/workflows/wai-deploy.yml

```diff
@@ -19,12 +19,13 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
+      - run: git config --global url."https://github.com/".insteadOf ssh://git@github.com/
```

Comment:
I tried to combine this with the credential setup on line 27. Couldn't figure it out so I'll just make the call twice.